app/.idea/gradle.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="GradleSettings">
+    <option name="linkedExternalProjectsSettings">
+      <GradleProjectSettings>
+        <option name="testRunner" value="PLATFORM" />
+        <option name="distributionType" value="DEFAULT_WRAPPED" />
+        <option name="externalProjectPath" value="$PROJECT_DIR$" />
+        <option name="gradleJvm" value="1.8" />
+        <option name="resolveModulePerSourceSet" value="false" />
```

Comment:
https://github.com/Sangmeebee/sangmeeCalculator/pull/1/commits/2179d0fb981071ef997dba26c1832aba44cad8fb
what is this??